docs/guides/Quick Start/02-product-management.md

```diff
@@ -0,0 +1,547 @@
+
+* name [string]
+* productNumber [string]
+* taxId [string]
+* price [object]
+* stock [int]
+
+The smallest required payload for a product can therefore be as follows:
+
+```json http
+{
+  "method": "post",
+  "url": "http://localhost/api/product",
+  "headers": {
+    "Content-Type": "application/json",
+    "Accept": "application/json",
+    "Authorization": "Bearer Your_API_Key"
+  },
+  "body": {
+    "name": "test",
+    "productNumber": "random",
+    "stock": 10,
+    "taxId": "a5da76b447db4d0aba62e6512dadf45b",
+    "price": [
+        {
+            "currencyId" : "b7d2554b0ce847cd82f3ac9bd1c0dfca", 
+            "gross": 15, 
+            "net": 10, 
+            "linked" : false
+        }
+    ]
+      }
+    }
+  }
+```
+
+```json json_schema
+{
+  "type": "object",
+  "description": "Parameters for product creation",
+  "properties": {
+    "name": {
+      "description": "Name of the product",
+      "type": "string"
+    },
+    "productNumber": {
+      "description": "Any random number given to product",
+      "type": "string"
+    },
+    "stock": {
+      "description": "Availability of stock",
+      "type": "string"
+    },
+    "taxId": {
+      "description": "ID of [tax](../../../adminapi.json/components/schemas/Tax)",
+      "type": "string"
+    },
+    "price": []
+      "currencyId": {
+      "description": "ID of [currency](../../../adminapi.json/components/schemas/Currency)",
+      "type": "string"
+      },
+      "gross": {
+      "description": "Gross price of a product",
+      "type": "string"
+      },
+      "net": {
+      "description": "Net price of a product",
+      "type": "string"
+      },
+      "linked": {
+      "description": "If set to true, net price is automatically calculated based on gross price and stored tax rate",
+      "type": "boolean"
+      }
+  }
+}
+```
+
+The `Productfeatureset` is a default template that extends from `productEntity` to insert all information for a new product.
+
+Here `linked` parameter takes boolean values. This signifies that if `priceForCurrency.linked` is true, the net price is calculated from the gross price based on tax type by the `calculatePrice()` method.
+
+The following payload examples contain UUIDs for various entities such as currencies, tax rates, manufacturers, or properties. These IDs are different on each system and must be adjusted accordingly.
+
+## Category
+
+All products are categorized in the catalogue. 
+
+Below is a sample request to create a category:
+
+```json http
+{
+  "method": "post",
+  "url": "http://localhost/api/category",
+  "headers": {
+    "Content-Type": "application/json",
+    "Accept": "application/json",
+    "Authorization": "Bearer Your_API_Key"
+  },
+  "body": {
+    "displayNestedProducts": true,
+    "type": "page",
+    "productAssignmentType": "product",
+    "name": "Home"
+    }
+  }
+```
+
+```json json_schema
+{
+  "type": "object",
+  "description": "Parameters for category creation",
```

Comment:
I think here it is fine to **not** specify a title, because you're not referring to the specific schema anywhere later in the text.